### DISCUSSION TITLE
Hide 0 Balance Accounts

### DISCUSSION THREAD

**Discussion by tribly** [ananthakumaran/paisa#182] [Ideas]
As per the latest update, there is a table with all checking balances on the frontpage. 
The table shows all accounts, even if they are at 0.00.

I have my debts/loans for friends under `Assets:Checking:Debts:<friendname>`, and for every friend, there is a table entry, even if it's 0,00.

Ledger for example doesn't show these accounts.

Would it be possible to add an option to hide them?

**Comment by ananthakumaran** [comment]
Won't this be hidden by default? (since balance only expands the first 3 levels)

**Comment by tribly** [comment]
Doesn't look like it's hidden by default.

![2024-02-14_17-06](https://github.com/ananthakumaran/paisa/assets/3883400/5af61abf-4c39-4b3f-b597-2da4db0c23d5)

**Reply by ananthakumaran** [reply]
I was thinking about assets > balance page, let me check what can be done here.

**Reply by tribly** [reply]
Under `Assets -> Balance` the 4th level is hidden, but `Assets:Checking:Debt:<name>` is shown.

**Comment by sdhawade** [comment]
I think a checkbox that does not show zero balance entries entirely will be useful as I have multiple stocks which end up causing a lot of noise since i ended up buying and selling from different PMS and smallcase.
